### DISCUSSION TITLE
Harvest seem not read custom.yaml file when update 23.08

### DISCUSSION THREAD

**Discussion by wyahn1219** [NetApp/harvest#2328] [Q&A] [answered]
Hello everyone
I configured it to get data about tiering by volume. ( Rest )
https://github.com/NetApp/harvest/discussions/2228

Harvest seems not read custom.yaml file when update 23.08
Is anything known about this issue?
```
[root@localhost harvest]#cat /var/log/harvest/*.log | grep custom
[root@localhost harvest]#
```

**Comment by rahulguptajss** [comment]
@wyahn1219 Could you please share below details.
1. What is the path of your custom.yaml?
2. Can you share your custom.yaml.
3. Also If you can email us harvest logs @ `ng-harvest-files@netapp.com`

**Comment by wyahn1219** [comment]
Hello @rahulguptajss 
Thank you for your help.
I sent the mail.

**Comment by rahulguptajss** [comment, marked as answer]
@wyahn1219 Based on the information provided in via email, were you successful in resolving the issue?

**Reply by wyahn1219** [reply]
Hello 
I apologize for the late reply.
I found the cause and resolved the issue.

my Harvest's use of zapi to obtain the metrics for ONTAP version 9.12.1. but I tried to modify the REST API configure file.
Thank you for your help

**Reply by cgrinds** [reply]
Thanks for the confirmation
